.travis.yml

```diff
@@ -12,7 +12,8 @@ before_script:
 - pip install black
 
 script:
-- pytest
+- pytest ml_tools/test_rectangle.py
+- pytest config/test_timewindow.py
```

Comment:
why do you have to run pytest against specific files instead of letting pytest discover them?